.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug Report
+about: Help us fix bugs in PlasmaPy
+
+---
+
+<!--
+
+Thanks for opening an issue! To help us react appropriately to your report,
+please first look through https://github.com/PlasmaPy/PlasmaPy/issues and check
+whether no other issue already describes your problem.
+
+For more general "how do I do X?" type questions, please speak to us in real
+time on https://riot.im/app/#/room/#plasmapy:matrix.org or ask on Discourse
+https://plasmapy.discourse.group
+
+-->
+
+## Reporting a bug
+
+<!--
+
+Before submitting a bug report please ensure that you can check off these boxes:
+
+-->
+
+- [ ] I am using the latest released version of PlasmaPy (most recent is visible in
+ the release notes (http://docs.plasmapy.org/en/stable/about/release_notes.html).
```

Comment:
We'll eventually need bug reports for long-term support versions too, but that probably won't be until we get to version 1.0.0 which is a while away.

Reply:
Yes, which means for now, let's keep support to recent versions :)